### PR TITLE
Don't call reload on order 

### PR DESCRIPTION
**Pull request by przemosk**
This PR introducing change from here: https://github.com/spree-contrib/spree_avatax_official/pull/151 + adjust specs 

Aparat from above changes I updated .travis file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
 
 language: ruby
-cache: bundler
 
-addons:
-  postgresql: 9.4
+rvm:
+  - 2.5.3
+  - 2.4.5
+  - 2.3.8
```
Comment (damianlegawiec) [inline .travis.yml:20]:
```suggestion
  - 2.6
  - 2.5
  - 2.4
  - 2.3
```

this way it will always use the newest versions

File: .travis.yml [hunk 2]
```diff
@@ -1,34 +1,37 @@
...
+  - DB=postgres
 
 gemfile:
   - gemfiles/spree_3_1.gemfile
   - gemfiles/spree_3_7.gemfile
-    #  - gemfiles/spree_3_2.gemfile
-    #  - gemfiles/spree_3_5.gemfile
+#  - gemfiles/spree_3_2.gemfile
+#  - gemfiles/spree_3_5.gemfile
   - gemfiles/spree_4_0.gemfile
```
Comment (damianlegawiec) [inline .travis.yml:31]: @przemosk please add 4.1 and 4.2 as well

Comment (przemosk) [reply .travis.yml:31]: @damianlegawiec sure, should I remove those commented versions ?

Comment (damianlegawiec) [reply .travis.yml:32]: @przemosk yes, let's remove the commented out

Comment (przemosk) [reply .travis.yml:30]: @damianlegawiec fixed

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**